includes/class-wc-stripe-api.php

```diff
@@ -16,6 +16,27 @@ class WC_Stripe_API {
 	const ENDPOINT           = 'https://api.stripe.com/v1/';
 	const STRIPE_API_VERSION = '2024-06-20';
 
+	/**
+	 * The Invalid API Keys cache key.
+	 *
+	 * @var string
+	 */
+	const INVALID_API_KEYS_CACHE_KEY = 'invalid_api_keys_rate_limit';
+
+	/**
+	 * The Invalid API Keys consecutive 401 error count threshold.
+	 *
+	 * @var int
+	 */
+	const INVALID_API_KEYS_COUNT_THRESHOLD = 5;
```

Comment:
This seems like a reasonable number, but we might want to increase this to a much higher number to cover the account cache refresh edge case. If the Account data cache expires, instantiating the UPE gateway will trigger ~25 API requests in a row.